elements/lisk-p2p/src/events.ts

```diff
@@ -40,12 +40,20 @@ export const EVENT_FAILED_TO_SEND_MESSAGE = 'failedToSendMessage';
 // Peer base
 export const REMOTE_SC_EVENT_RPC_REQUEST = 'rpc-request';
 export const REMOTE_SC_EVENT_MESSAGE = 'remote-message';
+
+// P2P Protocol messages
 export const REMOTE_EVENT_POST_NODE_INFO = 'postNodeInfo';
 export const REMOTE_EVENT_RPC_GET_NODE_INFO = 'getNodeInfo';
 export const REMOTE_EVENT_RPC_GET_PEERS_LIST = 'getPeers';
 export const REMOTE_EVENT_PING = 'ping';
 export const REMOTE_EVENT_PONG = 'pong';
 
+// P2P Protocol messages list
+export const PROTOCOL_REMOTE_EVENT_RCP_EVENTS: Set<string> = new Set([
```

Comment:
Didn't understand or relate this name with other names above. As per implementation detail it looks more like. `VALID_PROTOCOL_EVENTS` or `COUNTABLE_REMOTE_EVENTS` or `PROTOCOL_EVENTS_TO_RATE_LIMIT`